### DISCUSSION TITLE
Schedule next release?

### DISCUSSION THREAD

**Discussion by MarcinOrlowski** [logisim-evolution/logisim-evolution#1501] [Dev talk]
I think it'd be good to plan next release before new academical year started, what you think? Do we have any blockers (aside from PR pile)?

@maehne @BFH-ktt1 @davidhutchens @zdimension

**Comment by zdimension** [comment]
Good idea, were there any major/breaking changes [in the 238 commits since 3.7.2](https://github.com/logisim-evolution/logisim-evolution/compare/v3.7.2...develop)? IIRC, most of them were bugfixes, but I may have missed stuff.

**Comment by maehne** [comment]
I agree that we need a new release for the upcoming academic year. We should strive to address the issues assigned to the 3.8.0 milestone.
